docs/resources/aviatrix_spoke_gateway_subnet_group.md

```diff
@@ -0,0 +1,41 @@
+---
+subcategory: "Multi-Cloud Transit"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_spoke_gateway_subnet_group"
+description: |-
+  Creates and manages Aviatrix spoke gateway subnet groups
+---
+
+# aviatrix_spoke_gateway_subnet_group
+
+The **aviatrix_spoke_gateway_subnet_group** resource creates and manages the spoke gateway subnet groups.
+
+-> **NOTE:** This feature is only valid for Azure now.
```

Comment:
I think it sounds better if you remove "now". It would just be "This feature is only valid for Azure". Later if this feature is supported in other regions, we can just update the note to include other regions.